### PR TITLE
fix(CalloutBlock): hug contents is collapsing

### PR DESCRIPTION
**Pull request by fulopdaniel**
[From MDN docs:](https://developer.mozilla.org/en-US/docs/Web/CSS/container-type)

`Size containment turns off the ability of an element to get size information from its contents, which is important for container queries to avoid infinite loops.`

Meaning with container queries it is not possible to get the width of the content inside, so hugging the content width is not possible.

### CODE REVIEW

**Comment by ragi96** [comment]
Can we add a test to avoid regression? Something like adding content to a callout with hugWidth and then the width of the block should be XXpx?

**Review by ragi96 — COMMENTED** [review]
Works but as already stated a test to avoid regression would be good

**Comment by fulopdaniel** [comment]
> Works but as already stated a test to avoid regression would be good

Yep, added one

File: packages/callout-block/src/CalloutBlock.spec.ct.tsx [hunk 1]
```diff
@@ -266,4 +266,22 @@ describe('Callout Block', () => {
...
+                appearance: Appearance.Strong,
+                width: Width.HugContents,
+            },
+        });
+
+        mount(<CalloutBlockWithStubs />);
+
+        cy.get(CalloutBlockSelector).should(($el) => {
+            const width = parseFloat($el.css('width'));
+            expect(width).to.be.greaterThan(100);
```
Comment (ragi96) [inline packages/callout-block/src/CalloutBlock.spec.ct.tsx:284]: can we do this more exactly as its also true if its full width what also would be wrong

**Review by ragi96 — APPROVED** [review]

**Comment by ragi96** [comment]
Will merge and release it :)

**Pull request merged**